addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -13,6 +13,7 @@ weSnippetEditor.Class.include({
         .concat(['/website/static/src/xml/website.editor.xml']),
     events: _.extend({}, weSnippetEditor.Class.prototype.events, {
         'click .o_we_customize_theme_btn': '_onThemeTabClick',
+        'keyup .o_snipet_search_filter_input': '_onSearchFilterInputChange',
```

Comment:
- Use the `input` event, not `keyup`
- Add "Snippet" keyword in the method name, we may have other search filter inputs in the future